packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroup.tsx

```diff
@@ -11,6 +11,8 @@ export interface ChipGroupProps extends React.HTMLProps<HTMLDivElement> {
   children?: React.ReactNode;
   /** Additional classes added to the chip item */
   className?: string; 
+    /** Flag for having the chip group default to expanded */
+  isOpenOnInit?: boolean;
```

Comment:
Hi.  I was just talking to @jschuler  about the name of this prop.  We want to be consistent with the way we name our props across components.  The Page has a similar prop called 'defaultManagedSidebarIsOpen' so we think the chip Group should have a similar name.  How about 'defaultIsOpen`.  I will also rename the Page prop to defaultManagedSidebarIsOpen for consistency.

Reply:
Sounds good, thank you Titani -- I'll rename it accordingly